<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sorts</class>
 <widget class="QWidget" name="Sorts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonBubbleSort">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Bubble Sort</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberBubbleSort">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>6</number>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarSorts">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Bucket Sort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHeap">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Heap Sort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuick">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quick Sort</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberBucketSort">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberHeapSort">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberQuickSort">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushLoadArray">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar Arreglo</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCantidad">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>500000000</number>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetArrays">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetBubble">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetBucket">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetHeap">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetQuick">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMerge">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Merge Sort</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetMerge">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberMergeSort">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRun">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>85</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>GO!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
